### PR TITLE
feature/M2-2122-export-data-api

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,11 +27,13 @@
     "i18next": "^22.0.4",
     "i18next-browser-languagedetector": "^7.0.0",
     "javascript-time-ago": "^2.5.9",
+    "jszip": "^3.10.1",
     "lodash.debounce": "^4.0.8",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.uniqueid": "^4.0.1",
     "md-editor-rt": "2.8.1",
+    "objects-to-csv": "^1.3.6",
```
Comment (wildkiomi) [inline package.json:36]: do you use it?

Comment (kdanilenka32) [reply package.json:36]: later will be

File: src/modules/Dashboard/api/api.ts [hunk 2]
```diff
@@ -518,3 +503,11 @@ export const getAppletVersionChangesApi = (
   { appletId, version }: AppletVersionChanges,
   signal?: AbortSignal,
 ) => authApiClient.get(`/applets/${appletId}/versions/${version}/changes`, { signal });
+
+export const exportDataApi = ({ appletId, respondentId }: ExportData, signal?: AbortSignal) =>
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/api/api.ts:507]:
```suggestion
export const getExportDataApi = ({ appletId, respondentId }: ExportData, signal?: AbortSignal) =>
```

File: package.json (hunk 1 above)
Comment (wildkiomi) [reply package.json:36]: we have xlsx. it can work with csv. it doesn't suit you?

File: src/modules/Dashboard/api/api.ts (hunk 2 above)
Comment (kdanilenka32) [reply src/modules/Dashboard/api/api.ts]: fixed

File: package.json (hunk 1 above)
Comment (kdanilenka32) [reply package.json:36]: I will check later, for now it is hard to know

**Review by alex-i-e — APPROVED** [review]

**Pull request merged**